### PR TITLE
feat: add snyk code help pages

### PR DESCRIPTION
**Pull request by ArturSnyk**
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules

#### What does this PR do?
We would like to add help pages for `snyk code` command

### CODE REVIEW

**Review by neonnoon — APPROVED** [review]
> static code analysis scanning

Do we use this term elsewhere? To me it sounds like it should be either analysis or scanning, but not both?

File: help/commands-docs/code.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## SYNOPSIS
+
+`snyk` `code` \[<COMMAND>\] \[<OPTIONS>\] <PATH>
+
+## DESCRIPTION
+
+Find security issues using Static code analysis
+
+[For more information see the CLI for Snyk Code help page](https://docs.snyk.io/snyk-code/cli-for-snyk-code)
```
Comment (JackuB) [inline help/commands-docs/code.md:11]: This links to a 404

File: help/commands-man/snyk-sast.1 [hunk 2]
```diff
@@ -0,0 +1,95 @@
+.\" generated with Ronn-NG/v0.9.1
```
Comment (JackuB) [inline help/commands-man/snyk-sast.1:1]: Where is this file coming from? It should be `snyk-code.1` like above? A leftover file?

**Review by JackuB — CHANGES_REQUESTED** [review]

File: help/commands-docs/code.md (hunk 1 above)
Comment (neonnoon) [reply help/commands-docs/code.md:11]: We'll publish that page as soon as I merge this PR. Deadlock? 😄

File: help/commands-man/snyk-sast.1 (hunk 2 above)
Comment (neonnoon) [reply help/commands-man/snyk-sast.1]: You are right, nice catch!

**Review by JackuB — APPROVED** [review]

**Pull request merged**